contracts/external/cw-fund-distributor/src/testing/tests.rs

```diff
@@ -437,7 +436,7 @@ pub fn test_fund_cw20_zero_amount() {
         token_address,
         &cw20::Cw20ExecuteMsg::Send {
             contract: distributor_address.to_string(),
-            amount: Uint128::zero(),
+            amount: Uint128::zero(), // since cw20-base v1.1.0 this is allowed
```

Comment:
As commented here: since cw20-base v1.1.0, sending funds with zero amounts is allowed now.
@JakeHartnell @0xekez @NoahSaso